src/fragments/MessageListItem.tsx

```diff
@@ -33,6 +33,8 @@ const MessageListItem: React.FC<MessageListItemProps> = ({
 
   const [isHovering, hoverProps] = useHover({ mouseEnterDelayMS: 0 });
 
+  message
```

Comment:
looks like you probably didn't want this here.

Reply:
Yeah, I forgot about that, thanks for pointing out